assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/multiuser.properties

```diff
@@ -40,6 +40,10 @@ che.limits.workspace.env.ram=16gb
 # counts toward idleness.
 che.limits.workspace.idle.timeout=1800000
 
+# The length of time that a workspace will run, regardless of activity, before
+# the system will suspend it.
+che.limits.workspace.run.timeout=10800000
```

Comment:
@tomgeorge `che.limits.workspace.run.timeout`  Should be disabled by default 

@sleshchenko 

1. I was not aware of the fact that `che.limits.workspace.idle.timeout` can be set per-user. Can you point me to the docs if exist? In general, per-user settings for `che.limits.workspace.run.timeout` can be adressed in a separate issue if needed.

2. it is planned to be disabled by default and on Hosted Che set smth. like 5 / 8 h mainly to prevent abusers so  I do not think UX would be affected by this. In general, UX should be improved for vanilla idling in the first place

Reply:
@ibuziuk 

> 1. I was not aware of the fact

Fair point! Even that property comment does not mention that it's possible to override it per user with Resources API.
So, unfortunately that documentation is not ported from Che 6 but luckily I found it https://github.com/eclipse/che-docs/blob/2ca44011114cce9ef93963d633950289548371ce/src/main/pages/che-6/user-management/resource-management.adoc
The limits users have are just default values which can be overridden by Che Admin via REST API
![Screenshot_20200703_164313](https://user-images.githubusercontent.com/5887312/86475808-02c84700-bd4e-11ea-8430-29a5f5f14f8a.png)

> In general, per-user settings for che.limits.workspace.run.timeout can be adressed in a separate issue if needed.

I don't have any issues with that :+1: 

> 2. it is planned to be disabled by default and on Hosted Che set smth. like 5 / 8 h mainly to prevent abusers so I do not think UX would be affected by this. In general, UX should be improved for vanilla idling in the first place

Makes sense, I didn't dig deep enough to get the idea why we had that limit. If it's really big like 5-8h - then I agree that UX would be affected too much.